src/Twee/Friend/Visit friend.twee

```diff
@@ -40,7 +40,7 @@
 <<elseif $player.perversion.friend < 7 && $player.perversion.crossdressing > 5>>\
 	<<include [[Friend CD Catch-up 03]]>>\
 \
-<<elseif $player.perversion.friend < 8 && $player.perversion.crossdressing > 5>>\
+<<elseif $player.perversion.friend < 8 && $player.perversion.crossdressing > 7>>\

```

Comment:
I do not know if this leads to the intended behaviour, but it fits the pattern.

Reply:
I think this edit is close to correct.  The catch up scene should be triggering if the player skipped the park event, which is where friend perv is set to 8.  During that scene, PC's crossdress should be set to 7.  So, since there's only one possible event here instead of the usual 2, this should be >=7, not >7.

Reply:
```suggestion
<<elseif $player.perversion.friend < 8 && $player.perversion.crossdressing > 7>>\
<<elseif $player.perversion.friend < 8 && $player.perversion.crossdressing >= 7>>\
```